tools/topology/topology2/production/tplg-targets-ace2.cmake

```diff
@@ -17,4 +17,7 @@ SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
 SDW_JACK_OUT_STREAM=Playback-SimpleJack,SDW_JACK_IN_STREAM=Capture-SimpleJack"
 
+# No SDW Jack. SDW DMIC+SPK
+"cavs-sdw\;sof-lnl-rt1318-l12-rt714-l0\;PLATFORM=lnl,SDW_JACK=false,SDW_DMIC=1,\
+NUM_SDW_AMP_LINKS=2,SDW_DMIC_STREAM=SDW0-Capture"
```

Comment:
I don't add it because default Speaker Stream name on SDW1  has been in platform/intel/sdw-amp-generic.conf. 
Or still suggest to add for better readiness?


Define {
        SDW_SPK_STREAM 'SDW1-Playback'
        SDW_SPK_IN_STREAM 'SDW1-Capture'

Reply:
@macchian, if it is the default then it should not be added. I was just looking at the diff in the two definition in the cmake files.